react_thervu_application/src/control/input/input.component.js

```diff
@@ -3,27 +3,27 @@ import PropTypes from 'prop-types';
 import classes from '../input/input.css';
 
 const Input = (props) => {
-	return (
-		<div className={classes.inputControl}>
-			<input
-				id={props.id}
-				type={props.type}
-				value={props.value}
-				checked={props.checked}
-				onCheckbox={props.onCheckbox}
-				onChange={props.onChange}
-				placeholder={props.placeholder}/>
-		</div>
+  return (
+    <div className={classes.inputControl}>
+      <input
+        id={props.id}
+        type={props.type}
+        value={props.value}
+        checked={props.checked}
+        onCheckbox={props.onCheckbox}
```

Comment:
There is no synthetic event such as onCheckbox.. please verify it.

Reply:
okay, will check and change the synthetic event for checkbox